forPublication/vsm.md

```diff
@@ -1,42 +1,36 @@
+<br/><br/>
+Even though it stems from a manufacturing world, it can be applied to software as well.
+There it is an extremely helpful technique because of the high cost of of context switching in knowledge work and software development specifically.
+(link to blog the hidden cost of context switching) 
 #### How to do it for a software project?
-I recommend doing it in a workshop format where everyone who is involved in the value stream is gathered 
-(from inception to delivery of the feature to the customers). The technique is simple (but the devil is in the details).
+I recommend doing it in a workshop format where everyone who is involved in the value stream is gathered (from inception to delivery of the feature to the customers).
+The technique is simple (but the devil is in the details).
 Choose a few relevant and representative example slices to map the value stream horizontally from left to right. 
-Now vertically map the finer grained tasks including wait times and possible loops between them.<br/><br/>
-Next, classify each fine-grained task as Customer Value Adding (CVA), Non Value Adding(NVA) and Waste. This is harder 
-then it seems and expect a lot of healthy discussion. 
+Now vertically map the finer grained tasks including wait times and possible loops between them.
+<br/><br/>
+Next, classify each fine-grained task as Customer Value Adding (CVA), Non Value Adding(NVA) and Waste.
+This is harder then it seems and expect a lot of healthy discussion. 
 <br/><br/>
-Add the average processing and wait times for the item. It is now possible to calculate cycle efficiency! Do not
-be surprised if your cycle efficiency is below 30%! 
+Add the average processing and wait times for the item.
+It is now possible to calculate cycle efficiency!
+Do not be surprised if your cycle efficiency is below 30%! 
 <br/><br/>
-Note that the visual and collaboration aspects are important and performing these steps alone will often already lead 
-to ideas for optimizations. Next to that it can also bring an increased level of empathy in case of silo'ed organisations
- and you will help make the growth of a team more tangible. 
-
+Note that the visual and collaboration aspects are important and performing these steps alone will often already lead to ideas for optimizations.
+Next to that it can also bring an increased level of empathy in case of silo'ed organisations and you will help make the growth of a team more tangible. 
 #### Can I do it just for the development team?
-You can and it will be still be very useful for local optimization. The advantage is that a team can do it without any
-dependencies on the rest of the organisation. It is smaller and is easier to organise. Most value is 
-achieved by mapping the complete value stream though. If this is hard to organise in your project this could point to an 
+You can and it will be still be very useful for local optimization.
```

Comment:
I think at this point it is not clear what the alternative is for doing it just for the dev team.